### PR TITLE
Km doc updates

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

This PR updates several docs to the latest pipeline versions.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: website/docs/Pipelines/Single_Cell_ATAC_Seq_Pipeline/README.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ The main outputs of the scATAC workflow are the Snap file, Snap QC metrics, and
 | breakout_binCoordinates | Text file with the AM session ('Cell x bin accessibility' matrix) binChrom and binStart fields |
 | breakout_binCounts  | Text file with the AM session ('Cell x bin accessibility' matrix) idx, idy, and count fields |
 | breakout_barcodesSection  | Text file with the data from the BD session ('Barcode session' table) |
+| output_pipeline_version | Strong describing the version of the scATAC pipeline used. |
```
Comment (ekiernan) [inline website/docs/Pipelines/Single_Cell_ATAC_Seq_Pipeline/README.md:132]: Strong --> String

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 2]
```diff
@@ -118,6 +119,10 @@ The Paired-Tag workflow calls two WARP subworkflows and an additional task which
 | cell_calls_gex | `<input_id>_gex.emptyDrops` | TSV file containing the EmptyDrops results when the Optimus workflow is run in sc_rna mode. |
 | h5ad_output_file_gex | `<input_id>_gex.h5ad` | h5ad (Anndata) file containing the raw cell-by-gene count matrix, gene metrics, cell metrics, and global attributes. See the [Optimus Count Matrix Overview](../Optimus_Pipeline/Loom_schema.md) for more details. |
 | library_metrics | `<input_id>_library_metrics.csv` | Optional CSV file containing all library-level metrics calculated with STARsolo for gene expression data. |
+| multimappers_EM_matrix | `UniqueAndMult-EM.mtx` | Optional output produced when `soloMultiMappers` is "EM"; see STARsolo [documentation](https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md#multi-gene-reads) for more information.|
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:122]: Thanks for adding all of these!

**Review by ekiernan — APPROVED** [review]
Thank you so much, Kaylee

**Review by khajoue2 — APPROVED** [review]

**Review by kevinpalis — APPROVED** [review]
Looks great!

**Pull request merged**